modules/apps/batch-engine/batch-engine-service/src/test/java/com/liferay/batch/engine/internal/writer/ColumnValuesExtractorTest.java

```diff
@@ -218,14 +218,17 @@ private class ColumnDescriptorProviderImpl
 		@Override
 		public ColumnDescriptor[] getColumnDescriptors(
 			long companyId, String fieldName, int index,
-			ObjectValuePair<Field, Method> propertiesObjectValuePair,
+			Map<String, ObjectValuePair<Field, Method>> objectValuePairs,
```

Comment:
Now we have a map but we are just accesing to one of the values. Is this change necessary?

Reply:
Admittedly I don't understand the bigger picture of these changes but I fail to see how refactoring ObjectValuePair<Field, Method> propertiesObjectValuePair -> Map<String, ObjectValuePair<Field, Method>> objectValuePairs will prevent "leaking of object properties"

Reply:
Only objects has the concept of properties. If we ever need to implement this interface for other entities we will not have anything to pass there. The concept of `properties` is unique to objects and it should not leak in the interface. By passing all the value pairs we make it future proof.

Reply:
This was made evident when we renamed `properties` to `lazyProperties`. It made no sense to have that outside of the objects implementation.